robosuite/controllers/ee_imp.py

```diff
@@ -0,0 +1,199 @@
+from robosuite.controllers.base_controller import Controller
+from robosuite.utils.control_utils import *
+import robosuite.utils.transform_utils as T
+import numpy as np
+
+
+class EEImpController(Controller):
+    """
+    Controller for EE impedance
```

Comment:
More high-level description of the EE impedance controller

Reply:
Explain the input arguments (would be good to include some equations how they are used) -- and similar for the other controllers

Reply:
done. equation notes (as well as reference to the original [Khatib paper](http://khatib.stanford.edu/publications/pdfs/Khatib_1987_RA.pdf)) have been included in the `run_controller` method